### PR TITLE
Add some description for Hanamirc#file_options

### PR DESCRIPTION
**Pull request by dmitriy-strukov**

### CODE REVIEW

**Comment by dmitriy-strukov** [comment]
@jodosha  please review

**Comment by dmitriy-strukov** [comment]
@cllns can you please review ))?

File: lib/hanami/hanamirc.rb [hunk 1]
```diff
@@ -140,7 +140,12 @@ def symbolize(hash)
       Utils::Hash.new(hash).symbolize!
     end
 
+    # Checks file existence and return parsed values or empty hash
```
Comment (davydovanton) [inline lib/hanami/hanamirc.rb]: `return` -> `returns`

Comment (davydovanton) [reply lib/hanami/hanamirc.rb]: WDYT about this: `Returns options from hanamirc file`?

Comment (dmitriy-strukov) [reply lib/hanami/hanamirc.rb]: @davydovanton thank you for review! Yea I think it better `Returns options from hanamirc file`

**Pull request merged**